ricecooker/classes/nodes.py

```diff
@@ -226,14 +226,10 @@ def save_channel_children_to_csv(self, metadata_csv, topic_structure = list()):
             # Build out tag string
             current_level = self.title
             # prevent list from being passed by reference. Create new instance of list
-            structure_list = list(topic_structure)
-            # if len(structure_list) < 1:
-            #     structure_list = [current_level]
-            # else:
+            structure_list = list(structure_list)
```

Comment:
This line can completely be removed.

Reply:
I tried printing the structure list out with this line removed and it was printing out incorrectly. Seems that it was still passing in by reference

![Capture](https://user-images.githubusercontent.com/58611530/96502827-3482f200-1207-11eb-9ec8-472a0f43ce10.PNG)

Reply:
Ah, thanks, so it looks like what I read about `list()` as a function arg creating a new list each time was false. So let's make sure that any list or dict objects passed in default to `None` and that we set it when it is `None` the top of the function, like so:

```
if structure_list is None:
    structure_list = []
```

Reply:
updated list to be named as `branch_structure_list `for now as it is keeping track of the structure of each branch. Can name it to something different in the future